src/netplan.c

```diff
@@ -203,6 +203,66 @@ write_bond_params(yaml_event_t* event, yaml_emitter_t* emitter, const NetplanNet
+        || def->vxlan_params.max_fdb_entries
+        || def->vxlan_params.reduce_arp_proxy
+        || def->vxlan_params.l2_miss_notification
+        || def->vxlan_params.l3_miss_notification
+        || def->vxlan_params.route_short_circuit
+        || def->vxlan_params.udp_checksum
+        || def->vxlan_params.udp6_zero_checksum_tx
+        || def->vxlan_params.udp6_zero_checksum_rx
+        || def->vxlan_params.remote_checksum_tx
+        || def->vxlan_params.remote_checksum_rx
+        || def->vxlan_params.group_policy_extension
+        || def->vxlan_params.generic_protocol_extension
+        || def->vxlan_params.destination_port
+        || def->vxlan_params.port_range
+        || def->vxlan_params.flow_label
+        || def->vxlan_params.ip_do_not_fragment
+        || def->vxlan_params.independent) {
+        YAML_SCALAR_PLAIN(event, emitter, "parameters");
+        YAML_MAPPING_OPEN(event, emitter);
+        YAML_STRING(def, event, emitter, "remote", def->vxlan_params.remote);
+        YAML_STRING(def, event, emitter, "local", def->vxlan_params.local);
+        YAML_STRING(def, event, emitter, "group", def->vxlan_params.group);
+        YAML_UINT_0(def, event, emitter, "tos", def->vxlan_params.tos);
+        YAML_UINT_0(def, event, emitter, "ttl", def->vxlan_params.ttl);
+        YAML_BOOL_FALSE(def, event, emitter, "mac-learning", def->vxlan_params.mac_learning);
+        YAML_UINT_0(def, event, emitter, "fdb-ageing", def->vxlan_params.fdb_ageing);
+        YAML_UINT_0(def, event, emitter, "max-fdb-entries", def->vxlan_params.max_fdb_entries);
+        YAML_BOOL_FALSE(def, event, emitter, "reduce-arp-proxy", def->vxlan_params.reduce_arp_proxy);
+        YAML_BOOL_FALSE(def, event, emitter, "l2-miss-notification", def->vxlan_params.l2_miss_notification);
+        YAML_BOOL_FALSE(def, event, emitter, "l3-miss-notification", def->vxlan_params.l3_miss_notification);
+        YAML_BOOL_FALSE(def, event, emitter, "route-short-circuit", def->vxlan_params.route_short_circuit);
+        YAML_BOOL_FALSE(def, event, emitter, "udp-checksum", def->vxlan_params.udp_checksum);
+        YAML_BOOL_FALSE(def, event, emitter, "udp6-zero-checksum-tx", def->vxlan_params.udp6_zero_checksum_tx);
+        YAML_BOOL_FALSE(def, event, emitter, "udp6-zero-checksum-rx", def->vxlan_params.udp6_zero_checksum_rx);
+        YAML_BOOL_FALSE(def, event, emitter, "remote-checksum-tx", def->vxlan_params.remote_checksum_tx);
+        YAML_BOOL_FALSE(def, event, emitter, "remote-checksum-rx", def->vxlan_params.remote_checksum_rx);
+        YAML_BOOL_FALSE(def, event, emitter, "group-policy-extension", def->vxlan_params.group_policy_extension);
+        YAML_BOOL_FALSE(def, event, emitter, "generic-protocol-extension", def->vxlan_params.generic_protocol_extension);
+        YAML_UINT_0(def, event, emitter, "destination-port", def->vxlan_params.destination_port);
+        YAML_STRING(def, event, emitter, "port-range", def->vxlan_params.port_range);
```

Comment:
In which way would we specify a "port-range" value? Can you give an example for this?

It seems like NetworkManager uses two separate settings for this:
* source-port-max
* source-port-min

Combining those into a single "port-range" sounds nice, but I'm wondering about the syntax?

https://networkmanager.dev/docs/api/latest/nm-settings-nmcli.html#id-1.2.9.4.43

Reply:
In my testing, I could only specify an integer for `PortRange=` using systemd-networkd. Obviously that's not a "range." Documentation is lacking for this parameter in systemd-networkd.

Reply:
OK, I was able to get this to work as described. There isn't validation for it yet, though.